### PR TITLE
Basic field cleanup

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: src/widgets/forms/basic.ts [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+import { fieldParams } from './fieldParams'
+
+/**
+ * Create an anchor element with a label as the anchor text.
+ *
+ * @param dom The DOM
+ * @param property href for the anchor element
+ * @param fieldInQuestion field to produce a label for
+ *
+ * @ignore exporting this only for unit tests
```
Comment (megoth) [inline src/widgets/forms/basic.ts]: Any reason why you don't use @internal?

File: src/widgets/forms/basic.ts [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+ * @param predicate Predicate about which we want to find an editable RDF document
+ * @param def default RDF document to return if none found
+ *
+ * @ignore exporting this only for unit tests
+ */
+export function fieldStore (subject: Node, predicate: Node, def: Node | undefined): Node | undefined {
+  const sts = store.statementsMatching(subject, predicate)
+  if (sts.length === 0) return def // can used default as no data yet
+  if (
+    sts.length > 0 &&
```
Comment (megoth) [inline src/widgets/forms/basic.ts]: That's kinda implied by line 47, ain't it?

Comment (michielbdejong) [reply src/widgets/forms/basic.ts]: yeah, I didn't do any refactor in this branch. Now that we have full branch coverage, it's definitely something I want to do in a next PR!

File: src/widgets/forms/basic.ts (hunk 1 above)
Comment (michielbdejong) [reply src/widgets/forms/basic.ts]: ah right I keep forgetting which one is which :)

**Pull request merged**